boards/arm/stm32mp157c_dk2/Kconfig.defconfig

```diff
@@ -24,4 +24,13 @@ config UART_7
 
 endif # SERIAL
 
+if I2C
+
+config I2C_5
+	default y
+config I2C_5_IRQ_PRI
```

Comment:
We define IRQ priority in DT for STM32 I2C. Please remove it.

Reply:
Done with next push